src/main/java/com/box/sdk/BoxFileRequest.java

```diff
@@ -0,0 +1,434 @@
+package com.box.sdk;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Date;
+
+import com.eclipsesource.json.JsonObject;
+import com.eclipsesource.json.JsonValue;
+
+/**
+ * Represents a file request on Box.
+ */
+@BoxResourceType("file_request")
+public class BoxFileRequest extends BoxResource {
+
+    /**
+     * File Request URL Template.
+     */
+    public static final URLTemplate FILE_REQUEST_URL_TEMPLATE = new URLTemplate("file_requests/%s");
+    /**
+     * Copy File Request URL Template.
+     */
+    public static final URLTemplate COPY_FILE_REQUEST_URL_TEMPLATE = new URLTemplate("file_requests/%s/copy");
+
+    /**
+     * Constructs a BoxFileRequest for a file request with a given ID.
+     *
+     * @param api the API connection to be used by the file request.
+     * @param id  the ID of the file request.
+     */
+    public BoxFileRequest(BoxAPIConnection api, String id) {
+        super(api, id);
+    }
+
+    /**
+     * Gets information about this file request.
+     *
+     * @return info about this file request.
+     */
+    public BoxFileRequest.Info getInfo() {
+        URL url = FILE_REQUEST_URL_TEMPLATE.build(this.getAPI().getBaseURL(), this.getID());
+        BoxAPIRequest request = new BoxAPIRequest(this.getAPI(), url, "GET");
+        BoxJSONResponse response = (BoxJSONResponse) request.send();
+        JsonObject responseJSON = JsonObject.readFrom(response.getJSON());
+        return new Info(responseJSON);
+    }
+
+    /**
+     * Copies this file request that is already present on one folder, and applies it to another folder.
+     *
+     * <p>The only fields that will be copied are the ones that have been modified locally. For example, the following
+     * code won't copy any information (or even send a network request) since none of the info's fields were
```

Comment:
Just confirming - so the user will always need to make a change to the info, like the description or status? What happens if they want to copy a file request as-is, without any changes?